src/tests/ftest/util/test_utils.py

```diff
@@ -185,6 +188,108 @@ def create(self):
             "  Pool created with uuid %s and svc ranks %s",
             self.uuid, self.svc_ranks)
 
+    @fail_on(DaosApiError)
+    def create_dmg(self, dmg_bin_path, nsvc=None, group=None):
+        """Create a pool using dmg create pool.
+
+        1. Destroys the existing pool
+        2. Use dmg to create a pool
+        3. Create DaosPool object so that we can use other pool-related
+            operations
+        4. Set UUID of the new pool to the DaosPool object
+
+        Args:
+            dmg_bin_path (str): Directory where dmg is installed. Call
+                self.basepath + '/install/bin' in the test
+            nsvc (str): Number of pool service replicas. Defaults to None, in
+                which case 1 is used by the dmg binary in default. Currently
+                pool/pool_svc is the only test that uses this parameter.
+            group (str): Group name. Defaults to None, in which case the
+                system's username is passed into dmg_utils.py as group.
+
+        Returns:
+            Boolean: True if the pool create succeeds. False otherwise.
+        """
+        # 1. Destroys the existing pool
+        self.destroy_dmg(dmg_bin_path)
+        if self.target_list.value is not None:
+            self.log.info(
+                "Creating a pool on targets %s", self.target_list.value)
+        else:
+            self.log.info("Creating a pool")
+
+        # 2. Use dmg to create a pool
+        user = getpass.getuser()
+        if group == None:
+            group = user
+        # Currently, there is one test that creates the pool over the subset of
+        # the server hosts; pool/evict_test. To do so, the test needs to set
+        # the rank(s) to target_list.value starting from 0. e.g., if you're
+        # using 4 server hosts; wolf-1, wolf-2, wolf-3, and wolf-4, and want to
+        # create a pool over the first two hosts; wolf-1 and 2, then set the
+        # list [0, 1] to target_list.value. We'll convert it to the comma
+        # separated string and set it to dmg. For instance, [0, 1] will result
+        # in dmg pool create -r 0,1. If you don't set target_list.value, -r
+        # won't be used, in which case the pool is created over all the server
+        # hosts
+        if self.target_list.value == None:
+            ranks_comma_separated = None
+        else:
+            ranks_comma_separated = ""
+            for i in range(len(self.target_list.value)):
+                ranks_comma_separated += str(self.target_list.value[i])
+                # If this element is not the last one, append comma
+                if i < len(self.target_list.value) - 1:
+                    ranks_comma_separated += ","
+        self.log.info("ranks_comma_separated = %s" % ranks_comma_separated)
```

Comment:
(pylint-logging-not-lazy) Specify string format arguments as logging function parameters